include/sensors/core/tasks/capacitive_sensor_task.hpp

```diff
@@ -32,7 +32,7 @@ class CapacitiveMessageHandler {
     }
 
     void initialize() {
-        writer.write(DEVICE_ID_REGISTER, ADDRESS);
+        writer.write(0x0, ADDRESS, DEVICE_ID_REGISTER);
```

Comment:
this sort of adds to what I mean about the comments about `write` above - this is _really_ weird looking, and it would be nice if this could use either the old signature or a new signature for like `prep_read` that means you don't have to send data that will always be 0.